pkg/server/handler.go

```diff
@@ -63,7 +63,8 @@ func invalidateCookie(w http.ResponseWriter, name string) {
 func (ctrl *Controller) logoutHandler() http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		if r.Method != "POST" && r.Method != "DELETE" {
-			ctrl.writeErrorMessage(w, http.StatusMethodNotAllowed, "only POST and DELETE are allowed")
+			renderServerError(w, "you can only logout via a POST or DELETE")
+			ctrl.log.Error("you can only logout via a POST or DELETE")
 			return
```

Comment:
`renderServerError` has been removed not so long ago. I guess, 405 code (`http.StatusMethodNotAllowed`) is more appropriate here.

We also should avoid method name strings and use `http.MethodPost` and `http.MethodGet` instead:

```go
func (ctrl *Controller) logoutHandler() http.HandlerFunc {
	return func(w http.ResponseWriter, r *http.Request) {
		switch r.Method {
		case http.MethodPost, http.MethodGet:
			invalidateCookie(w, jwtCookieName)
			http.Redirect(w, r, "/login", http.StatusTemporaryRedirect)
		default:
			ctrl.writeErrorMessage(w, http.StatusMethodNotAllowed, "only POST and DELETE methods are allowed")
		}
	}
}

```